int32 robot_id
int32 counter
int32 state
float32 yaw
move_base_msgs/MoveBaseGoal position
multi_robot_sl/wind_information wind_information
multi_robot_sl/alco_concentration alco_concentration